src/main.js

```diff
@@ -170,9 +170,11 @@ class MiniGraphCard extends LitElement {
     const additional = conf.hours_to_show > 24 ? { day: 'numeric', weekday: 'short' } : {};
     conf.format = { hour12: !conf.hour24, ...additional };
 
+    // override points per hour to mach group_by function
     if (conf.group_by === 'date') {
-      // override points per hour to mach 1 point per day (24h)
       conf.points_per_hour = 1 / 24;
+    } else if (conf.group_by === 'hour') {
+      conf.points_per_hour = 1;
```

Comment:
consider switch instead - easier to extend in the future and better readability IMO

Reply:
Replaced